db/migrate/20191008024334_remove_unused_from_users.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUnusedFromUsers < ActiveRecord::Migration
```

Comment:
Might be good idea to name it `RemoveUnusedColumnsFromUsers`. Or to follow existing convention: `RemoveColumn1Column2FromTableName`.

Reply:
Should be better now.